README.md

```diff
@@ -305,7 +305,7 @@ docker login ${REGISTRY} -u ${UPBOUND_ACCOUNT_EMAIL}
 Build package.
 
 ```console
-up xpkg build --name package.xpkg --ignore ".github/workflows/*,hack/*"
+up xpkg build --name package.xpkg --package-root=package
```

Comment:
I like the direction but we were also implicitly including `examples` before, so maybe we can do
```suggestion
up xpkg build --name package.xpkg --package-root=package --examples-root="examples"
```

Reply:
Done!